source/serialization/account_link.rst

```diff
@@ -15,8 +15,9 @@ Account Key Link
 AccountKeyLinkTransaction
 =========================
 
+This transaction is required for all accounts willing to activate delegated harvesting.
```

Comment:
This transaction is required for all accounts wanting to activate delegated harvesting.